app/main/views/suppliers.py

```diff
@@ -361,3 +359,66 @@ def set_selection_questions(supplier_id, framework_slug):
         abort(400, "Database Error: {}".format(e))
 
     return jsonify(selectionAnswers=supplier_framework.serialize()), status_code
+
+
+@main.route('/suppliers/<supplier_id>/frameworks/interest', methods=['GET'])
```

Comment:
I guess how we use it on the frontend is up for grabs to an extent?  It feels right to list all relevant frameworks to me - e.g. say there is a framework that has just closed and two frameworks that are currently open, and the supplier has interest registered in two of them - in that case a single page might show three different contingent messages, and it would be better for the frontend to be able to make a single API call to find all the relevant frameworks, rather than one for each potential framework.

But I'm prepared to be convinced otherwise by a counter-example.